app/src/main/java/com/github/steroidteam/todolist/view/dialog/ItemListener.java

```diff
@@ -0,0 +1,6 @@
+package com.github.steroidteam.todolist.view.dialog;
+
+public interface ItemListener {
+
+    void onClick();
+}
```

Comment:
Method 'onClick()' is never used.

You used DialogInterface.OnClickListener instead of this Interface. So I think you can remove it.
But nice, I didn't know that we can do it this way, I will maybe do some refactoring to use it in List/Note selection fragment.